core/services/ssh_user.rb

```diff
@@ -6,42 +6,65 @@
 # The module creates a new user on the VM.
 module SshUser
   def self.save_to_file(info, configuration)
-    File.open(Configuration.ssh_user_path(configuration), 'w') { |f| f.write(JSON.generate(info)) }
+    info.each do |node_name, user_name|
+      next if user_name.nil?
+
+      IO.write(
+        Configuration.ssh_user_file(configuration, node_name),
```

Comment:
Can user creation be combined with other recipes? I.e. can this command be implemented in partial as install-product command?